catalogs/glade-na-cordex.json

```diff
@@ -0,0 +1,41 @@
+{
+  "esmcat_version": "0.1.0",
+  "id": "glade-na-cordex",
+  "description": "This is an ESM collection for NA-CORDEX data accessible on NCAR's GLADE disk storage system in /glade/collections/cdg/data/cordex/data",
+  "catalog_file": "path/to/glade-na-cordex.csv.gz",
+  "assets": {
+    "column_name": "path",
+    "format": "netcdf"
+  },
+  "attributes": [
+    { "column_name": "variable", "vocabulary": "" },
+    { "column_name": "scenario", "vocabulary": "" },
+    { "column_name": "driver", "vocabulary": "" },
+    { "column_name": "rcm", "vocabulary": "" },
+    { "column_name": "frequency", "vocabulary": "" },
+    { "column_name": "grid", "vocabulary": "" },
+    { "column_name": "biascorrection", "vocabulary": "" },
+    { "column_name": "common", "vocabulary": "" }
```

Comment:
I am curious... What does the common attribute mean?

When looking at the csv, I am noticing that it is unique across all entries except where it is missing:

```python
In [8]: df = pd.read_csv("intake-esm-datastore/catalogs/glade-na-cordex.csv.gz")                                                 

In [9]: df.head()                                                                                                                
Out[9]: 
                                                path variable scenario      driver  ... frequency     grid biascorrection  common
0  /glade/collections/cdg/data/cordex/data/raw/NA...     prec     hist  MPI-ESM-MR  ...       ann  NAM-22i            raw  common
1  /glade/collections/cdg/data/cordex/data/raw/NA...      vas     hist  MPI-ESM-MR  ...       ann  NAM-22i            raw  common
2  /glade/collections/cdg/data/cordex/data/raw/NA...  sfcWind     hist  MPI-ESM-MR  ...       ann  NAM-22i            raw  common
3  /glade/collections/cdg/data/cordex/data/raw/NA...      uas     hist  MPI-ESM-MR  ...       ann  NAM-22i            raw  common
4  /glade/collections/cdg/data/cordex/data/raw/NA...      tas     hist  MPI-ESM-MR  ...       ann  NAM-22i            raw  common
In [12]: df.common.unique()                                                                                                                                                   
Out[12]: array(['common', nan], dtype=object)
```

Reply:
It indicates whether or not the data has been interpolated to a standard lat-lon grid.  All the files on the NAM-22i and NAM-44i grids lines up, so you can combine them using join_new().  The files on the NAM-22 and NAM-44 (no 'i' at the end) grids use a map projection that varies from RCM to RCM, so they can't be combined.

The counterpart to "common" would be "native", but my understanding from when we discussed it before was that to aggregate selectively based on the value, you'd want a value and missing.